components/skus/browser/rs/cxx/Cargo.toml

```diff
@@ -12,7 +12,7 @@ futures = "0.3.19"
 libc = { version = "0.2" }
 skus = { path = "../lib" }
 serde_json = "1.0"
-tracing-subscriber = { version = "0.2.0", default-features = false, features = ["ansi", "fmt", "registry"] }
+tracing-subscriber = { version = "0.2.0", default-features = false, features = ["ansi", "ansi_term", "fmt", "registry", "sharded-slab", "thread_local"] }
```

Comment:
For cargo, `ansi` implies `ansi_term` and `registry` implies `shared-slab` and `thead_local`. If these aren't making it into BUILD.gn that's probably a `gnrt` bug. 

But they do no harm here, so ok.

Reply:
they are all explicit in gn so I just copied that list here